### PR TITLE
A few corrections on "collaborate"

### PR DESCRIPTION
**Pull request by alix-tz**
As I re-read on last time #48 , I spotted a couple of errors so here is a PR to fix them quickly.

Also, I figured you could make the information on how to stop sharing a little bit more visible.

### CODE REVIEW

File: docs/walkthrough_collaborate.md [hunk 1]
```diff
@@ -31,23 +31,20 @@ Only the owner of a team is allowed to add or remove members or to transfer owne
...
 You can find more information about inviting new users in the [user-related features walkthrough](walkthrough_users.md#invite) section.
 
 ## Share a project or a document
 
-You can share projects and documents to others users of the same eScriptorium instance.
+You can share projects and documents with others users of the same eScriptorium instance.
 
 ### Share a project
 
-You can share an opened project you own by clicking on the "Share this project" button, located in the project dashboard at {base_url}/project/{project_slug}/documents/. **Only the owner of a project can share it.** It will then open a pop-up box where you have two options:
```
Comment (alix-tz) [inline docs/walkthrough_collaborate.md:44]: I wasn't sure why you put "an opened project" here?

Comment (HugoSchtr) [reply docs/walkthrough_collaborate.md:44]: It's just because the button is accessible from an opened project! :)

**Comment by HugoSchtr** [comment]
Thanks for the corrections! I'm merging :+1:

File: docs/walkthrough_collaborate.md (hunk 1 above)
Comment (alix-tz) [reply docs/walkthrough_collaborate.md:44]: what do you mean by "opened project"?

**Pull request merged**